cerbero/build/cookbook.py

```diff
@@ -390,6 +394,12 @@ def _load_recipes(self):
                     st.touch()
                 else:
                     self.reset_recipe_status(recipe.name)
+            # check that we use the same config md5
+            # Use getattr as config_hash was added later
+            saved_config_hash = getattr(st, 'config_hash', 0)
```

Comment:
I don't recall how deserialization works with new fields, but this could be avoided if config_hash was a class attribute and therefore always initialized to a sane value, same for the other ones.